src/main/resources/assets/tfc/patchouli_books/book/en_us/entries/getting_started/predators.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "Predators",
+    "icon": "minecraft:textures/items/bone.png",
+    "category": "getting_started",
+	"advancement": "tfc:main/stone_age",
+	"sortnum": 6,
+    "pages": [
+        {
+            "type": "text",
+            "text": "There are many predators in TFC for you to hunt and be hunted by. Some attack by day, some at night. If you can slay one, you'll be rewarded with plentiful meat and $(l:processes/leatherworking)Hide$()."
+		},
+		{
+			"type": "entity",
```

Comment:
May want to include basic information about each animal? e.g. one more page saying if they hunt at day vs. at night, where you'll find them etc.